src/quick-start.md

```diff
@@ -67,6 +70,8 @@ This is a quick overview of the attributes of the `AdmissionPolicy` resource:
 * `operations`: what operations for the previously given types should
   be forwarded to this admission policy by the API server for
   evaluation.
+* `mutating`: a boolean value that must be set to `true` for policies that can
```

Comment:
Shouldn't `mutating` flagged as `optional` ?

Reply:
We made it mandatory because, if a mutating policy is reported a validation only, the kubernetes API will choke on the mutation requests.

I'm not particularly happy about the amount of information a user has to write into our CR declaration, especially because many of these details can be embedded into the metadata of the `.wasm` policy file. We have plans to improved that once we have more time.